app/src/main/java/net/skyscanner/backpack/demo/compose/SelectStory.kt

```diff
@@ -0,0 +1,174 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import net.skyscanner.backpack.compose.fieldset.BpkFieldStatus
+import net.skyscanner.backpack.compose.select.BpkSelect
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.SelectComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
+
+@Composable
+@SelectComponent
+@ComposeStory
+fun SelectStory(modifier: Modifier = Modifier) {
+    Column(modifier) {
+        Row(
+            modifier = Modifier.padding(BpkSpacing.Base),
+            horizontalArrangement = Arrangement.Start,
+        ) {
+            DefaultSelectSample()
+        }
+        Row(
+            modifier = Modifier.padding(BpkSpacing.Base),
+            horizontalArrangement = Arrangement.Start,
+        ) {
+            DisabledSelectSample()
+        }
+        Row(
+            modifier = Modifier.padding(BpkSpacing.Base),
+            horizontalArrangement = Arrangement.Start,
+        ) {
+            ErrorSelectSample()
+        }
+    }
+}
+
+@Composable
+@SelectComponent
+@ComposeStory("TextBox")
```

Comment:
nitpick on naming - `TextBox` makes it sounds like you can enter text. maybe `Select box only`

Reply:
nice.